SpringProject/src/main/java/com/gradproject2019/users/exception/InvalidRegexFormatException.java

```diff
@@ -0,0 +1,8 @@
+package com.gradproject2019.users.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.BAD_REQUEST, reason="Invalid registration credentials.")
+public class InvalidRegexFormatException extends RuntimeException {
```

Comment:
The regex is something you define in your code, if it's invalid that means that you defined it wrong in your code. I think you meant InvalidCredentialsFormatException.